easybuild/easyconfigs/z/ZeroMQ/ZeroMQ-4.2.5-fosscuda-2018b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'ZeroMQ'
+version = '4.2.5'
+
+homepage = 'http://www.zeromq.org/'
+description = """ZeroMQ looks like an embeddable networking library but acts like a concurrency framework.
+ It gives you sockets that carry atomic messages across various transports like in-process,
+ inter-process, TCP, and multicast. You can connect sockets N-to-N with patterns like fanout,
+ pub-sub, task distribution, and request-reply. It's fast enough to be the fabric for clustered
+ products. Its asynchronous I/O model gives you scalable multicore applications, built as asynchronous
+ message-processing tasks. It has a score of language APIs and runs on most operating systems."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = ['https://github.com/zeromq/libzmq/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['cc9090ba35713d59bb2f7d7965f877036c49c5558ea0c290b0dcc6f2a17e489f']
+
+# --with-pgm will use shipped OpenPGM (in foreign subdir)
+configopts = '--with-pic --with-system-pgm '
+configopts += 'OpenPGM_CFLAGS="-I$EBROOTOPENPGM/include/pgm-${EBVERSIONOPENPGM%.*}" '
+configopts += 'OpenPGM_LIBS="-L$EBROOTOPENPGM/lib -lpgm -lrt -lpthread -lm" '
+
+dependencies = [
+    ('OpenPGM', '5.2.122'),
+    ('util-linux', '2.32.1'),
```

Comment:
@akesandgren This needs to be downgraded to `2.32` to make the dependency variants test pass...